src/proxy-ui-api/frontend/src/store/modules/notifications.ts

```diff
@@ -0,0 +1,142 @@
+import { ActionTree, GetterTree, Module, MutationTree } from 'vuex';
+import { RootState } from '../types';
+
+
+export interface NotificationsState {
+  successMessageCode: string;
+  successMessageRaw: string;
+  showSuccessCode: boolean;
+  showSuccessRaw: boolean;
+  errorMessageCode: string;
+  errorMessageRaw: string;
+  errorObject: any;
+  showErrorObject: boolean;
+  showErrorRaw: boolean;
+  showErrorCode: boolean;
+}
+const getDefaultState = () => {
+  return {
+    successMessageCode: '',
+    successMessageRaw: '',
+    showSuccessCode: false,
+    showSuccessRaw: false,
+    errorMessageCode: '',
+    errorMessageRaw: '',
+    showErrorObject: false,
+    showErrorCode: false,
+    showErrorRaw: false,
+    errorObject: undefined,
+  };
+};
+
+// Initial state. The state can be reseted with this.
+const notificationsState: NotificationsState = getDefaultState();
+
+export const getters: GetterTree<NotificationsState, RootState> = {
+  showSuccessCode(state): boolean {
```

Comment:
Isn't this state of type NotificationsState from above? I think types should be used always when possible - it's a lot easier for other developers to get into the implementation.